build/project-template/__PROJECT_NAME__.xcodeproj/project.pbxproj

```diff
@@ -209,7 +209,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cd \"$SRCROOT/metadataGenerator/bin\"\nbash metadata-generation-build-step.sh";
+			shellScript = "cd \"$SRCROOT/metadataGenerator/bin\"\n./metadata-generation-build-step.py";
```

Comment:
I think we should remove the file extension from the script in order to future-proof it, in case we end up rewriting it in Ruby or something. Справка: https://github.com/NativeScript/ios-runtime/issues/308#issuecomment-140673487